lib/rust/parser/src/syntax/statement.rs

```diff
@@ -273,29 +278,31 @@ fn parse_statement<'s>(
     let private_keywords = scan_private_keywords(&line.items);
     let start = private_keywords;
     let items = &mut line.items;
-    if let Some(annotation) = try_parse_annotation(items, start, precedence) {
+    let mut parsed = None;
+    parsed = parsed.or_else(|| {
```

Comment:
We could just directly assign `try_parse_annotation`, couldn't we?

Reply:
Calling it the same way as the other parsers expresses that it isn't special, it just happens to be tried first.